shared/utility/nusight/NUhelpers.hpp

```diff
@@ -27,11 +27,9 @@
 
 namespace utility::nusight {
 
-    namespace {
+    using message::support::nusight::DataPoint;
 
-        using message::support::nusight::DataPoint;
-
-        constexpr float TIMEOUT = 2.5;
+    namespace helpers {
```

Comment:
`buildGraph` should be inside an anon namespace and `is_iterable` should be moved to a header in the type_traits folder.

Reply:
There is a clang-tidy check (which we have enabled) that warns on anonymous namespaces.

Reply:
and I'll leave the type trait change to you, for a further PR :p

Reply:
Which check is this, so I can look into it.

Reply:
https://clang.llvm.org/extra/clang-tidy/checks/google-build-namespaces.html

[relevant guideline](https://google.github.io/styleguide/cppguide.html#Internal_Linkage)

[relevant reason provided by CMU](https://wiki.sei.cmu.edu/confluence/display/cplusplus/DCL59-CPP.+Do+not+define+an+unnamed+namespace+in+a+header+file#:~:text=Do%20not%20define%20an%20unnamed%20namespace%20in%20a%20header%20file&text=When%20an%20unnamed%20namespace%20is,used%20within%20that%20translation%20unit.)

Reply:
Yeah your right.

Reply:
#532